ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/representative-steps-order.factory.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { RepresentativeJourneySteps } from './representative-journey-steps';
+
+@Injectable()
+export class RepresentativeStepsOrderFactory {
```

Comment:
Do we even have a mobile journey for representative? If not, maybe we can remove the steps factory and join it into the representative class? Less code, less tests.

Reply:
Make sense